index.rst

```diff
@@ -2,24 +2,54 @@
 
 .. sectnum::
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
+Abstract
+========
 
-.. TODO: Delete the note below before merging new content to the main branch.
+This documents describes the sequence of events that led to the network failure during the 202212A run in auxtel.
 
-.. note::
+Description of the Incident
+===========================
 
-   **This technote is a work-in-progress.**
+This incident initiated between 8:00-8:30 CLT/3:00-3:30PST December, 6 2022, during the first night of the 2022A run on the Auxiliary Telescope.
+We experienced a catastrophic network failure that took down the network in the entire observatory.
+The failure extended to both the internal and external network, both over the WiFi and the cable connections.
+All WiFi connections were dropped and we could no longer connect to the network.
 
-Abstract
-========
+Up until the issue started we were preparing to go on sky with the Auxiliary Telescope.
+The TMA was experiencing power issues and would not be able to operate during the night for the soak and start tracker testing. 
+Alysha Shugart and Tiago Ribeiro were executing some dome movement tests with the remote help of Bruno Quint.
 
-This documents describes the sequence of events that led to the network failure during the 202212A run in auxtel.
+At one point around 8:30 PM local time, Tiago Ribeiro went to update LOVE in the love02 node with the latest version of the logging system for testing during the night.
```

Comment:
You may want to mention that you were doing this remotely from the US

Reply:
I was actually in the console in Chile with everyone else.. :)